vizro-core/mkdocs.yml

```
@@ -45,7 +45,7 @@ nav:
           - Custom components: pages/user-guides/custom-components.md
           - Custom actions: pages/user-guides/custom-actions.md
           - Custom figures: pages/user-guides/custom-figures.md
-  - API Reference:
+  - Reference:
```

I think I'd rather call it API Reference or just API, but these docs are really just meant as an API lookup. Reference seems too broad?

```suggestion
  - API
```

---

I am following the diataxis model, but thinking about it, our readers don't know that, so yes, let's go back to API reference. Thanks for making me reconsider!